system-int-test/src/cmr/system_int_test/utils/url_helper.clj

```diff
@@ -691,6 +691,11 @@
   []
   (format "http://localhost:%s/acls" (transmit-config/access-control-port)))
 
+(defn access-control-permissions-url
```

Comment:
This can be removed as we will add the tests in access-control app.

Reply:
Removed